staging/src/k8s.io/apiserver/pkg/util/flowcontrol/metrics/timing_ratio_histogram.go

```diff
@@ -70,6 +70,7 @@ func NewTimingRatioHistogram(opts *TimingRatioHistogramOpts) *TimingRatioHistogr
 
 // NewTestableTimingHistogram adds injection of the clock
 func NewTestableTimingRatioHistogram(nowFunc func() time.Time, opts *TimingRatioHistogramOpts) *TimingRatioHistogram {
+	//nolint:govet // copylocks: assignment copies lock value to ratioedOpts: k8s.io/component-base/metrics.TimingHistogramOpts contains sync.Once contains sync.Mutex
```

Comment:
is this copy a real issue? cc @MikeSpreitzer

Reply:
I think I traced this back up the call chain and it looked like it was only used in test data where copying the mutex doesn't matter.

Reply:
But as this is an exported function, it's not possible to be sure about that. Perhaps it can and/or should be in a `_test.go` file?

Reply:
not sure, maybe @MikeSpreitzer can weigh in

Reply:
So what's the conclusion? Leave it as-is, with the govet suppression?

Reply:
I am OK with that.

Reply:
Please disregard my earlier comment about this being only for testing: `NewTestableTimingRatioHistogram` (despite the name) is the implementation of `NewTimingRatioHistogram` and that is a public API of the package.

We would have to remove everything (`NewTimingRatioHistogram`, `NewTestableTimingRatioHistogram`, and the test of the code).